database/migrations/2020_07_30_142212_add_membership_contract_to_users_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddMembershipContractToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function(Blueprint $table) {
```

Comment:
It this can't be fixed due to time constraints, so be it, but I don't like that the membership contract is linked to a user. We have the membership object for this, so Ideally the contract is linked to the membership, and memberships get a "pending" state where they are not yet valid until a board member approves the membership. In this case, what happens if a user stops their membership and joins as a member afterwards? Will the previous signature still be valid? What if the terms have changed?